### PR TITLE
Added documentation about computed properties

### LINKED ISSUE #2301: FRONTEND WITH FRAMEWORK: VueJS-2.3 => Computed Properties in Vue.js
### Description

Contributors are required to explain about Computed Properties in Vue.js, which should cover questions like:-
What is Computed Properties in Vue.js?
When do we have to use this?
How does this work?
How to implement Computed Properties with code (also give theory explanation of code)?
You can take 2-3 examples with codes of different types to explain this.
etc....

**Note:**

Plagiarism is strictly prohibited.
Changes should be made inside the Frontend_Web_Development_React_Angular_Vue/ dir inside Frontend_Web_Development_React_Angular_Vue branch.
Files need to be added into the Week2 folder only.

### Domain

Frontend Dev React/Angular/Vue

### Type of Contribution

Documentation

### Code of Conduct

- [X] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.

### PR DESCRIPTION
**Pull request by Safwan310**
<hr>

## Description 📜

This documentation explains computed properties in Vue.js
Fixes #2301

<hr>

## Type of change 📝

- [x] Documentation (Content Creation in the form of codes or tutorials)

<hr>

## Domain of Contribution 📊

- [x] Frontend Dev React/Angular/Vue

<hr>

## Checklist ✅

- [x] I follow [Contributing Guidelines](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CONTRIBUTING.md) & [Code of conduct](https://github.com/girlscript/winter-of-contributing/blob/main/.github/CODE_OF_CONDUCT.md) of this project.
- [x] I have performed a self-review of my own code or work.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [x] My changes generates no new warnings.
- [x] I'm GWOC'21 contributor

<hr>

## Screenshots / Gif (Optional) 📸

<hr>

### CODE REVIEW

**Comment by Safwan310** [comment]
@anandhu720 can you please verify my PR

**Comment by SourabhSaraswat-191939** [comment]
@Safwan310, 
As per content, it is ok. But I think examples are not sufficient.
Like your example is showing that computed property uses cache, and that is correct. But there is one more huge case that you mentioned in theory but not in code.  That is computed properties are called automatically when their dependencies change.

Don't remove your current example, let it be there for basics. Just give one more example covering computed properties with dependencies change.
You can take examples like ->. You have two input boxes and a line just below that input boxes which will be showing the text "The sum of values in two input boxes => {{result}}"
So whenever a value changes in any of the input boxes, computed property will calculate it and put it at the place of {{result}}


You can use the link mentioned below for reference, here is the example of showing full name when the first name and last name change is shown. So, by this you will get an idea that how you can implement my "Sum of Numbers example"
**Reference** ->  https://www.tutorialspoint.com/vuejs/vuejs_computed_properties.htm

**Comment by Safwan310** [comment]
> @Safwan310,
> As per content, it is ok. But I think examples are not sufficient.
> Like your example is showing that computed property uses cache, and that is correct. But there is one more huge case that you mentioned in theory but not in code. That is computed properties are called automatically when their dependencies change.
> 
> Don't remove your current example, let it be there for basics. Just give one more example covering computed properties with dependencies change.
> You can take examples like ->. You have two input boxes and a line just below that input boxes which will be showing the text "The sum of values in two input boxes => {{result}}"
> So whenever a value changes in any of the input boxes, computed property will calculate it and put it at the place of {{result}}
> 
> You can use the link mentioned below for reference, here is the example of showing full name when the first name and last name change is shown. So, by this you will get an idea that how you can implement my "Sum of Numbers example"
> **Reference** -> https://www.tutorialspoint.com/vuejs/vuejs_computed_properties.htm

Sure Sourabh, I will add another example with the mentioned changes

**Comment by Safwan310** [comment]
@SourabhSaraswat-191939 I have made the necessary changes, please take a look.

**Comment by Safwan310** [comment]
> @SourabhSaraswat-191939 I have made the necessary changes, please take a look.

👀

**Comment by SourabhSaraswat-191939** [comment]
> @SourabhSaraswat-191939 I have made the necessary changes, please take a look.

Sorry for the late reply @Safwan310,
The changes look good to me.

We are almost ready to merge. There is just a small change required.

File: Frontend_Web_Development_React_Angular_Vue/Vue/Week 2/computed_properties.md [hunk 1]
```diff
@@ -0,0 +1,160 @@
...
+    <title>Document</title>
+    <!-- We are using a CDN to use Vue.js -->
+    <script src="https://cdn.jsdelivr.net/npm/vue@2.6.14/dist/vue.js"></script>
+</head>
+<body>
+    <div id = "demo" >
+        <!-- This div container has two input fields. The value in these input fields will be
+        assigned to the numberOne and numberTwo data properties in the vue instance.  -->
+        <div style="display: flex; flex-direction: row;justify-content: space-between; padding: 200px;">
+            <input style="width: 33%; height:50px;" type="number" v-model = "numberOne" placeholder="x">
```
Comment (SourabhSaraswat-191939) [inline Frontend_Web_Development_React_Angular_Vue/Vue/Week 2/computed_properties.md:126]: As binding is not covered yet. So please mention that.
"We are using 2-way binding with the help of v-model, which we will cover later in this Documentation"

Comment (SourabhSaraswat-191939) [reply Frontend_Web_Development_React_Angular_Vue/Vue/Week 2/computed_properties.md:126]: Please make this change. Then we are good to go with your PR.

**Comment by Safwan310** [comment]
@SourabhSaraswat-191939 I have added that small change.

**Comment by SourabhSaraswat-191939** [comment]
@Safwan310, LGTM 👍🏻 

@anandhu720, we can merge this PR.

**Review by anandhu720 — APPROVED** [review]

**Pull request merged**